tests/discovery/testrendezvous.nim

```diff
@@ -0,0 +1,176 @@
+{.used.}
+
+# Nim-Libp2p
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+import sequtils, strutils
+import chronos
+import ../../libp2p/[protocols/rendezvous, switch]
+import ../../libp2p/discovery/discoverymngr
+import ../helpers
+import ./utils
+
+suite "RendezVous":
+  teardown:
+    checkTrackers()
+
+  asyncTest "Request locally returns 0 for empty namespace":
+    let (nodes, rdvs) = setupNodes(1)
+    nodes.startAndDeferStop()
+
+    const namespaceEmpty = ""
```

Comment:
maybe move consts to suite/file level:  namespaceEmpty, namspaceFoo, namespaceBar...